improver/calibration/reliability_calibration.py

```diff
@@ -542,31 +818,41 @@ def __repr__(self):
         result = "<ApplyReliabilityCalibration: minimum_forecast_count: {}>"
         return result.format(self.minimum_forecast_count)
 
-    @staticmethod
-    def _threshold_coords_equivalent(forecast, reliability_table):
-        """Ensure that the threshold coordinates are identical in the
-        reliability table and in the forecast cube. If not raise an
-        exception.
+    def _extract_matching_reliability_table(self, forecast, reliability_table):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=6001543)